diff --git a/dist/cli/build/bundle-server.js b/dist/cli/build/bundle-server.js
index 95a1a0a73326004b794d711874ccd12242bdb18e..88f4eec4fedfa35124fca1eef49db04a21a64b90 100644
--- a/dist/cli/build/bundle-server.js
+++ b/dist/cli/build/bundle-server.js
@@ -133,7 +133,7 @@ export async function bundleServer(buildOpts, projectOpts) {
         banner: {
             // We need to import them here, assigning them to `globalThis` does not work because node:timers use `globalThis` and thus create an infinite loop
             // See https://github.com/cloudflare/workerd/blob/d6a764c/src/node/internal/internal_timers.ts#L56-L70
-            js: `import {setInterval, clearInterval, setTimeout, clearTimeout, setImmediate, clearImmediate} from "node:timers"`,
+            js: `import {setInterval, clearInterval, setTimeout, clearTimeout} from "node:timers"`,
         },
         platform: "node",
     });
